src/Consolonia.Core/Drawing/DrawingContextImpl.cs

```diff
@@ -210,7 +210,8 @@ public void DrawGlyphRun(IBrush foreground, IGlyphRunImpl glyphRun)
                 return;
             }
 
-            var shapedBuffer = (ShapedBuffer)glyphRunImpl.GlyphInfos;
+            var shapedBuffer = glyphRunImpl.GlyphInfos as ShapedBuffer;
```

Comment:
Cast was better? with "as" there is no difference between null and different type.

Reply:
the linter was bitching about it.

Reply:
Do we know which rule is that? I will suppress it better.

Reply:
I don't.

Reply:
Ok, then next time just ping me please at the moment it happens, would be easier.
I would investigate that.